package.json

```diff
@@ -28,6 +28,7 @@
 		"node": ">= 10.12.0"
 	},
 	"dependencies": {
+		"@faucet-pipeline/marx": "0.0.1",
```

Comment:
That doesn't inspire confidence?

We have a responsibility to existing users, so gotta make sure that everything continues to work as before. That's why I'd also suggest we do _not_ delete the test here.